Raspi/Test/das_data_generator.py

```diff
@@ -9,14 +9,22 @@ def send_fake_data(send_data_func, duration, rate, immitate_teensy=False):
         we don't want to send that data """
     start_time = round(time.time(), 2)
     gps_speed = 0
+    # MHP Workshop location
+    gps_lat = -37.908756
+    gps_long = 145.13404
     while True:
         current_time = round(time.time(), 2)
         total_time = round(current_time - start_time, 2)
 
         # Generate data message
-        data = "gps=1&gps_lat=00&gps_long=00&gps_alt=00&gps_course=00&"
+        data = "gps=1"
+        data += "&gps_lat=" + str(gps_lat)
+        data += "&gps_long=" + str(gps_long)
+        gps_lat += random.randint(0, 20) * 0.00001
```

Comment:
I would leave a comment stating why you chose those values. (eg. Random distance up to x metres away etc.)

Reply:
I added some comments and changed the code to do movement based on metres instead of an arbitrary degree.